### ISSUE TITLE
CreationDate metadata shows non-reproducible string representation of object instead of data (using sphinx)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rst2pdf/rst2pdf#525]
```
The people working on reproducible builds on Debian found out that the 
monkey-patched PDFDate does not extend PDFObject, so Reportlab formats it using 
str instead of .format(), resulting in a string like 
"<rst2pdf.createpdf.PDFDate instance at 0x7fc7cd56b320>" instead of a proper 
date, which gets embedded in the CreationDate PDF metadata.

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=777251

I've checked that this happens when running rst2pdf from sphinx, but not when 
running it directly.

The attached patch comes directly from the debian bugreport.

Please answer the following questions, they are *very* important:

* Version of rst2pdf? 0.93

* Version of reportlab? 3.1.8

* Are you using rst2pdf with Sphinx, and if true, what version? 1.2.3

```

Original issue reported on code.google.com by `elena.va...@gmail.com` on 9 Feb 2015 at 1:20

Attachments:
* [patch.diff](https://storage.googleapis.com/google-code-attachments/rst2pdf/issue-525/comment-0/patch.diff)

**Comment by valholl** [comment]
Would turning this patch into a pull request help?

**Comment by pmaupin** [comment]
Sure, I'd review a pull request.  Haven't gone back to look at all the bugs...

**Comment by oz123** [comment]
fixed in #551
